src/diffusers/loaders.py

```diff
@@ -87,6 +90,9 @@ def map_from(module, state_dict, *args, **kwargs):
 
 
 class UNet2DConditionLoadersMixin:
+    text_encoder_name = TEXT_ENCODER_NAME
```

Comment:
Think this is the best option we have right now! 

Just FYI: Feel slightly uncomfortable with the unet having to know about the text encoder name here as we now super lightly entangle those concepts now, but I also think that this is the best design at the moment! Thank for adding it!

Reply:
Do you want me to add the names to the utils constants?